detalhes-pt-br.md

```diff
@@ -14,11 +14,11 @@ Inspired by you, the community, through your actions and stories.
 
 We ask all participants to read through the details to ensure that the global community is working toward a shared goal. Thank you for honoring the values and following the rules of participation.
 
-## Participation details
+## Detalhes de participação
 
-- Hacktoberfest is open to everyone in our global community. Whether you’re a seasoned contributor or looking for projects to contribute to for the first time, you’re welcome to participate.
-- Pull requests can be made in any GitHub-hosted repositories/projects. As long as the project is public and GitHub-hosted, your pull requests will count toward your participation.
-- You can sign up anytime between October 1 and October 31. Just be sure to sign up on the official Hacktoberfest website for your pull requests to count.
+- A Hacktoberfest está aberta a todos em nossa comunidade global. Seja você um contribuidor experiente ou esteja procurando projetos para contribuir pela primeira vez, sinta-se à vontade para participar.
+- As solicitações de pull request podem ser feitas em qualquer repositório ou projeto hospedado no GitHub. Contanto que o projeto esteja público e hospedado no GitHub, suas solicitações de pull request contarão para sua participação.
+- Você pode se inscrever a qualquer momento entre 1º de outubro e 31 de outubro. Apenas certifique-se de se inscrever no site oficial do Hacktoberfest para que suas solicitações de pull request sejam contabilizadas.
```

Comment:
Tem como deixar as palavras em inglês em itálico? só para a gente seguir padrão?